### PR TITLE
Add Docker image for Alpine with NGINX Plus

### PR DESCRIPTION
**Pull request by lucacome**

### CODE REVIEW

**Comment by lucacome** [comment]
@pleshakov added the docs

File: build/Dockerfile [hunk 1]
```diff
@@ -12,8 +12,21 @@ RUN apt-get update \
...
 FROM nginx:1.19.9-alpine AS alpine
 
-RUN apk add --no-cache libcap \
-	&& rm -rf /var/cache/apk/*
+RUN apk add --no-cache libcap
+
+
+############################################# Base image for Alpine with NGINX Plus #############################################
+FROM alpine:3.13 as alpine-plus
+ENV NGINX_PLUS_VERSION 24-r1
```
Comment (pleshakov) [inline build/Dockerfile:20]: currently, the other Plus images all have R23. Could we have one version of N+ across all images?

Comment (lucacome) [reply build/Dockerfile:20]: 3.13 is only supported in R24.

Comment (pleshakov) [reply build/Dockerfile:20]: would it make sense to upgrade other images to R24 first then?

Comment (lucacome) [reply build/Dockerfile:20]: Created #1604 to update to R24

Comment (pleshakov) [reply build/Dockerfile:20]: @lucacome will we able to not use a specific patch version of N+ for Apline similarly to Debian in #1604?

Comment (lucacome) [reply build/Dockerfile:20]: yep I'll update this PR once the other one is done

**Review by pleshakov — APPROVED** [review]

**Pull request merged**